pyincore/utils/cge_ml_file_util.py

```diff
@@ -2,158 +2,160 @@
-        # logger.info(f"factor: {factor} - model_coeff shape: {model_coeffs[factor].shape}")
-
-    return model_coeffs, sectors, base_cap_sector_order
-
-
-def parse_csv(file_name: str, sectors: List[str]) -> np.ndarray:
-    """parse_csv Utility function to parse the csv files
-
-    Parameters
-    ----------
-    file_name : str
-        Path to the csv file
-
-    Returns
-    -------
-    np.ndarray
-        Numpy array containing the data from the csv file
-    """
-
-    logger.info(f"Parsing {file_name}")
-    df = pd.read_csv(file_name)
-
-    col_name = df.columns[1]
-    val_col = df.columns[2]
-
-    # Convert 'sectors' column to categorical with the desired order
-    df[col_name] = pd.Categorical(df[col_name], categories=sectors, ordered=True)
-
-    # Sort the DataFrame based on the 'names' column
-    df = df.sort_values(by=col_name)
-
-    return np.array(df[val_col], dtype=np.float64)
-
-
-def parse_base_vals(
-    filenames: List[str],
-    ds_sectors: List[str],
-    base_cap_sector_order: Dict[str, List[str]],
-) -> Tuple[List[np.ndarray], np.ndarray]:
-    """parse_base_vals parse_base_vals will parse the base values from the input file and return them as numpy arrays
-
-    Parameters
-    ----------
-    filenames : List[str]
-        Paths to the .csv files containing the base values. It has to be organized this order, starting from:
-        1. Domestic Supply
-        2. Gross Income
-        3. Household Count
-        4. Factor Demand
-        5. Base Capital
-
-    Returns
-    -------
-
-    Tuple[np.ndarray, np.ndarray]
-        Tuple containing two numpy arrays. The first array is the base capital for different factors and the second is the base capital
-    """
-
-    base_cap_factors: List[np.ndarray] = []
-    base_cap: np.ndarray = parse_csv(filenames[-1], ds_sectors).reshape(
-        1, -1
-    )  # 1 x K array K = number of sectors in the model
-    # logger.info(f"base_cap shape: {base_cap.shape}")
-
-    for filename, sector_order in zip(filenames[:-1], base_cap_sector_order.values()):
-        base_cap_factors.append(
-            parse_csv(filename, sector_order).reshape(-1, 1)
-        )  # k_i x 1 array k_i = number of sectors k for a factor i
-        # logger.info(f"{os.path.basename(filename)} shape: {base_cap_factors[-1].shape}")
-
-    return base_cap_factors, base_cap
-
-
-def parse_files(
-    model_filenames: Dict[str, str], filenames: List[str]
-) -> Tuple[Dict[str, List[str]], np.ndarray, np.ndarray, Dict[str, np.ndarray]]:
-    """parse_files Utility function to parse the input files
-
-    Parameters
-    ----------
-    model_filenames : Dict[str, str]
-        Dictionary containing the factor name and the path to the coefficient file
-    filenames : List[str]
-        Paths to the .csv files containing the base values. It has to be organized this order, starting from:
+class CGEMLFileUtil:
```

Comment:
You need to add another blank line before the class name, I see pep8 error about there only being one line between the class name and the logger = pyglobals.LOGGER line